open-craft/opencraft
# Make prospector play well with pylint again

**Description**

Current version of prospector does not work with pylint 1.6.* (pylint warnings are silently suppressed).

This PR reverts pylint to 1.5.6 and fixes pylint violations that have accumulated while pylint wasn't working.

It also globally disables the `no-member` check, which is very unreliable and therefore not helpful.

For more info see https://tasks.opencraft.com/browse/OC-2143

**Testing Instructions**

1. Checkout this branch, update dependencies (`pip install -r requirements.txt`).
2. Run `make test_prospector`. It should report no violations.
3. Intentionally make some pylint violations (add an unused import somewhere, remove an existing `# pylint: disable=xxx` comment, etc...)
4. Run `make test_prospector` again, just to ensure that it correctly reports the violations you just made.

**Reviewer**:
